plugins/file/plugin.go

```diff
@@ -21,16 +26,70 @@ func init() {
 	plugins.RegisterPlugin("file", setupFile6, setupFile4)
 }
 
+//Record holds an IP lease record
+type Record struct {
+	IP      net.IP
+	expires int64
+}
+
 // StaticRecords holds a MAC -> IP address mapping
 var StaticRecords map[string]net.IP
 
 // DHCPv6Records and DHCPv4Records are mappings between MAC addresses in
 // form of a string, to network configurations.
 var (
 	DHCPv6Records map[string]net.IP
-	DHCPv4Records map[string]net.IP
+	DHCPv4Records map[string]Record
```

Comment:
Can this be a map of a reference to `Record` instead?

Reply:
Yes.